src/codegen/templates/library_interpreter/default_c_function_call.py.mako

```diff
@@ -7,18 +7,24 @@
     )
     from codegen.utilities.text_wrappers import wrap, docstring_wrap
 
-    function_call_args = generate_interpreter_function_call_args(function)
+    function_call_args, dateTime_args = generate_interpreter_function_call_args(function)
```

Comment:
PEP8 says variables are snake_case.

However, this is moot because I think you should delete dateTime_args.